cmd/integration/integration.go

```diff
@@ -75,9 +75,9 @@ var (
 	// Limit the number of concurrent tests.
 	maxConcurrency int
 
-	longTestTimeout = time.Second * 300
+	longTestTimeout = time.Second * 500
```

Comment:
Me neither, but we also create some services and I have no idea what kind of thing ingress is doing behind the scenes.